fundamentals/apple/gettingstarted/final/Favourites/Shared/FavouriteNumberView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  FavouriteNumberView.swift
+//  Shared
+//
+//  Created by Peter Friese on 05.07.22.
+//  Copyright © 2021 Google LLC. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import Combine
+import FirebaseAnalytics
+import FirebaseAnalyticsSwift
+
+class FavouriteNumberViewModel: ObservableObject {
+  @Published var favouriteNumber: Int = 42
```

Comment:
nit: The type can be implied here (number literals default to int if they don't have a decimal component iirc)